egcg_core/clarity.py

```diff
@@ -46,8 +46,8 @@ def get_valid_lanes(flowcell_name):
         artifact = flowcell.placements.get(placement_key)
         if not artifact.udf.get('Lane Failed?', False):
             valid_lanes.append(lane)
-    valid_lanes = sorted(valid_lanes)
-    app_logger.info('Valid lanes for %s: %s', flowcell_name, str(valid_lanes))
+    valid_lanes.sort()
+    app_logger.debug('Valid lanes for %s: %s', flowcell_name, valid_lanes)
```

Comment:
Ought this be ', '.join(valid_lanes) to make it look better, or does debugger do that?

Reply:
`valid_lanes` is a list of ints here, so the call would have to be `', '.join(str(l) for l in valid_lanes)` - not really worth it for a debug statement

Reply:
Okay thanks for that, was just wondering!